src/app/backend/dashboard.go

```diff
@@ -86,7 +89,17 @@ func main() {
 	// TODO(maciaszczykm): Move to /appConfig.json as it was discussed in #640.
 	http.Handle("/api/appConfig.json", handler.AppHandler(handler.ConfigHandler))
 	http.Handle("/metrics", prometheus.Handler())
-	log.Print(http.ListenAndServe(fmt.Sprintf("%s:%d", *argBindAddress, *argPort), nil))
+
+	// Listen for http and https
+	addr := fmt.Sprintf("%s:%d", *argBindAddress, *argPort)
+	if argBindAddress.Equal(net.IPv4(127, 0, 0, 1)) {
```

Comment:
@cheld thanks for pointing it our, I've done changes like this:

1.  `--port` => `--insecure-port`, and the new `--port` will be used for secure port.
1.  `--bind-address` => `--insecure-bind-address`, and the new `--bind-address` will be used for secure address.
1.  update the `serve` task to use `--insecure-port` to keep compatibility.